server/controllers/cohort.js

```diff
@@ -157,9 +134,7 @@ const getCohortDetails = (req, resp) => {
     .exec()
     .then(doc => {
       if (!doc) {
-        throw new NotFoundException(
-          `Data of cohort id: ${cohortId} not found.`
-        );
+        throw new NotFoundException();
```

Comment:
there is no more 'then' in the chain, this could be simple `return resp.sendStatus(400);`